framework_tvb/tvb/core/neocom/_registry.py

```diff
@@ -29,10 +29,21 @@
 #
 
 import typing
+from functools import partial
+
+from tvb.adapters.datatypes.db.projections import ProjectionMatrixIndex
+from tvb.adapters.datatypes.db.sensors import SensorsIndex
+from tvb.adapters.datatypes.db.surface import SurfaceIndex
+from tvb.adapters.datatypes.h5.projections_h5 import ProjectionMatrixH5
+from tvb.adapters.datatypes.h5.sensors_h5 import SensorsH5
+from tvb.adapters.datatypes.h5.surface_h5 import SurfaceH5
 from tvb.basic.neotraits.api import HasTraits
 from tvb.core.entities.model.model_datatype import DataType
 from tvb.core.neotraits.h5 import H5File
 from tvb.core.neotraits.db import HasTraitsIndex
+from tvb.datatypes.projections import make_proj_matrix, ProjectionMatrix
+from tvb.datatypes.sensors import make_sensors
+from tvb.datatypes.surfaces import make_surface
```

Comment:
These specific imports here are not nice. 
They are breaking the module dependencies we want to achieve.
Could these go into tvb.config.init.data_registry.py ?